src/components/Navigation/Navigation.js

```diff
@@ -5,13 +5,15 @@ import {
   toggleDisplayModal,
   toggleJoinSurveyModal,
 } from '../../state/actions/displayModalAction';
+import { useOktaAuth } from '@okta/okta-react';
 import 'antd/dist/antd.css';
 
 import { Layout, Menu } from 'antd';
 
 const { Header, Content, Footer } = Layout;
 
 const Navigation = props => {
+  const { authState, authService } = useOktaAuth();
```

Comment:
You've destructured `authState` however you're not using it. Are you sure you need to include this to achive the logout functionality for the app?